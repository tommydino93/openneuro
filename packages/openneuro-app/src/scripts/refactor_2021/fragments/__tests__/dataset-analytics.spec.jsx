import React from 'react'
import { shallow } from 'enzyme'
import DatasetAnalytics from '../dataset-analytics.jsx'

describe('DatasetAnalytics component', () => {
  it('renders with common props', () => {
    const wrapper = shallow(<DatasetAnalytics downloads={5} views={50} />)
    expect(wrapper).toMatchSnapshot()
  })
  it('renders for snapshots', () => {
    const wrapper = shallow(
      <DatasetAnalytics downloads={5} views={50} snapshot />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
