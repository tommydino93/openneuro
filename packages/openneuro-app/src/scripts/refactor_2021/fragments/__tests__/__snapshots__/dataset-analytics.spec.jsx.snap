// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatasetAnalytics component renders for snapshots 1`] = `
<Fragment>
  <Styled(span)>
    <Metric
      display={true}
      snapshot={true}
      type="downloads"
      value={5}
    />
  </Styled(span)>
  <Styled(span)>
    <Metric
      display={true}
      snapshot={true}
      type="views"
      value={50}
    />
  </Styled(span)>
</Fragment>
`;

exports[`DatasetAnalytics component renders with common props 1`] = `
<Fragment>
  <Styled(span)>
    <Metric
      display={true}
      type="downloads"
      value={5}
    />
  </Styled(span)>
  <Styled(span)>
    <Metric
      display={true}
      type="views"
      value={50}
    />
  </Styled(span)>
</Fragment>
`;
